src/libs/xapi-stats.ts

```diff
@@ -0,0 +1,438 @@
+import defaults from 'lodash/defaults.js'
+import findKey from 'lodash/findKey.js'
+import forEach from 'lodash/forEach.js'
+import identity from 'lodash/identity.js'
+import map from 'lodash/map.js'
```

Comment:
Si on utilise `lodash-es`:

```suggestion
import { defaults, findKey, forEach, identity, map } from 'lodash-es'
```